src/main/java/org/dcsa/ovs/service/impl/TransportEventServiceImpl.java

```diff
@@ -2,21 +2,23 @@
 
 import lombok.RequiredArgsConstructor;
 import org.dcsa.core.service.impl.ExtendedBaseServiceImpl;
+import org.dcsa.ovs.model.OperationsEvent;
 import org.dcsa.ovs.model.TransportEvent;
 import org.dcsa.ovs.repository.TransportEventRepository;
-import org.dcsa.ovs.service.transportEventService;
+import org.dcsa.ovs.service.OperationsEventService;
```

Comment:
OperationsEventService is not needed here

Reply:
removed